roles/kubernetes/master/defaults/main.yml

```diff
@@ -88,3 +88,8 @@ kubeadm_token_ttl: 0
 
 ## Variable for influencing kube-scheduler behaviour
 volume_cross_zone_attachment: false
+
+## Encrypting Secret Data at Rest
+encrypt_secret_data: false
```

Comment:
Please prefix your variables with `kube_`
These should be kube_encrypt_secret_data, kube_encrypt_token, and kube_encryption_algorithm.